DanielSantos01/money-map-frontend
# Feat/summary

What I did:
- Summary screen

How to test:
1. git checkout feat/summary
2. git pull origin feat/summary
3. yarn install
4. yarn start
5. Access the app and click on the profile picture
6. Check the summary screen
